package.json

```diff
@@ -39,8 +40,16 @@
     "bluebird": "^3.4.6",
     "bluebird-co": "^2.2.0",
     "child-process-promise": "^2.2.0",
+    "five-bells-integration-test": "github:interledgerjs/five-bells-integration-test#master",
+    "five-bells-service-manager": "github:interledgerjs/five-bells-service-manager#master",
```

Comment:
You should not add `five-bells-integration-test` and `-service-manager`. The loader [installs them automatically](https://github.com/interledgerjs/five-bells-integration-test-loader/blob/master/src/bin/integration-loader#L12-L16). Very handy if you write new integration tests that depend on changes in the service-manager.